docs/user_manual.md

```diff
@@ -443,26 +445,7 @@ This tells the operator to use the value "zone2" as the fault domain for every p
 
 When running across multiple KCs, you will need to apply more care in managing the configurations to make sure all the KCs converge on the same view of the desired configuration. You will likely need some kind of external, global system to store the canonical configuration and push it out to all of your KCs. You will also need to make sure that the different KCs are not fighting each other to control the database configuration. You can set different flags in `automationOptions` to control what the operator can change about the cluster. You can use these fields to designate a master instance of the operator which will handle things like reconfiguring the database.
 
-When upgrading a cluster, you will need to bounce instances simultaneously across all of your KCs. The best way to do this is to disable killing processes through `automationOptions`, and then kill them outside of the reconciliation loop. You can determine if each instance of the operator is ready for the kill by checking the `generations` field in the cluster status. The new generation will appear there under the field `needsBounce` when the operator is ready to do a bounce, but is forbidden from doing so by the automation options.
-
-Example with automation options disabled:
-
-    apiVersion: apps.foundationdb.org/v1beta1
-    kind: FoundationDBCluster
-    metadata:
-        name: sample-cluster
-    spec:
-      version: 6.2.20
-      databaseConfiguration:
-        storage: 5
-      automationOptions:
-        configureDatabase: false
-        killProcesses: false
-      faultDomain:
-        key: foundationdb.org/kubernetes-cluster
-        value: zone2
-        zoneIndex: 2
-        zoneCount: 5
+You should always specify an `instanceIDPrefix` when deploying an FDB cluster to multiple Kubernetes clusters. You should set it to a different value in each Kubernetes cluster. This will two multiple instances in different Kubernetes cluster from using the same instance ID.
```

Comment:
```suggestion
You must always specify an `instanceIDPrefix` when deploying an FDB cluster to multiple Kubernetes clusters. You must set it to a different value in each Kubernetes cluster. This will prevent instance ID duplicates in the different Kubernetes clusters.
```